research/other/permissions.md

```diff
@@ -24,26 +24,30 @@ There are multiple user journeys that require the set up and management of permi
 
 Permissions are a critical and enabling feature of the FNHS platform and were the subject of three rounds of user interviews during June, August and November 2021. Overall, despite some minor usability issues during setup (which has been described as clunky) and around restricting content assignment (which has been described as complex), managers considered permissions extremely valuable to help them restrict access in their desired way - and in some cases, permissions set Kahootz apart from other tools (for example Sharepoint.) 
 
-Kahootz updated the platform in August 2022 to address some known issues with permissions, but no user research has been conducted since this time, therefore it is recommended that an expert review is conducted at the earliest opportunity. It is anticipated that the navigational support provided around the key user journeys will need to be reviewed and assessed to identify and prioritise improvements.
+Kahootz updated the platform in August 2022 to address some known issues with permissions, but no user research has been conducted since this time, therefore it is recommended that an expert review (with a content focus) is conducted at the earliest opportunity. It is anticipated that the navigational support provided around the key user journeys will need to be reviewed and assessed to identify and prioritise improvements.
 
-Several design concepts were also tested with users to inform designs for FNHS 2.0, particularly in respect to the central admin area or dashboard. These concepts should be reviewed again in terms of the learnings they contribute to the Kahootz platform.
+Several design concepts were also tested with users to inform designs for FNHS 2.0, particularly in respect to the central admin area or dashboard. These concepts should be reviewed again in terms of the learnings they may contribute to the Kahootz platform.
 
-No new designs were implemented into the product FNHS 2.0 as the designs were scaled back for MVP. 
+No new designs were implemented into the product FNHS 2.0, as the designs were scaled back for MVP. 
  
 
 ## Recommendations  
 
 ### Short term (in Kahootz)  
 
-Review the user journeys for set up and management of permissions with a primary focus on updating content to improve clarity around the processes. In parallel, review the navigation for these user journeys to improve consistency and remove confusing duplication.
+* Review the user journey for set up of permissions with a primary focus on updating the website content to improve clarity around the process and the choices involved. In parallel, assess the set up flow to check if a preview function could be added as a quick win. 
```

Comment:
...around the process and the **choices** involved: replace choices with options
and after involved add: so that users can make informed choices. In general users should be encouraged to work in the open as much as possible.